net/respondd/src/respondd.c

```diff
@@ -292,19 +290,20 @@ static void add_provider(const char *name, const struct respondd_provider_info *
 	*pos = pentry;
 }
 
-static void load_providers(void) {
+static void load_providers(const char *path) {
 	update_time();
 
-	/* Maximum number of request types, might be made configurable in the future */
-	if (!hcreate_r(32, &htab)) {
-		perror("hcreate_r");
-		exit(EXIT_FAILURE);
+	int cwdfd = open(".", O_DIRECTORY);
```

Comment:
I would have loved to open with `O_PATH` here, but I wasn't able to get the build environment to recognize the option, even when I included all headers and feature macros that the manpage mentioned.